RuntimeUnityEditor/Windows/ObjectTree/ObjectTreeViewer.cs

```diff
@@ -172,8 +175,18 @@ private void DisplayObjectTreeHelper(GameObject go, int indent, ref int currentC
                 }
                 GUILayout.EndHorizontal();
 
-                if (needsHeightMeasure && Event.current.type == EventType.repaint)
-                    _singleObjectTreeItemHeight = Mathf.CeilToInt(GUILayoutUtility.GetLastRect().height);
+                if (Event.current.type == EventType.repaint)
+                {
+                    if (needsHeightMeasure)
+                    {
+                        _singleObjectTreeItemMargin = GUI.skin.label.margin.top;
+                        _singleObjectTreeItemHeight = GUILayoutUtility.GetLastRect().height + _singleObjectTreeItemMargin;
+                    }
+                    if (SelectedTransform == go.transform && _scrollTreeToSelected)
```

Comment:
Moved this down here because this was causing an issue. GetLastRect was getting the position of the element prior to the one we want. Was causing an issue if the first item was the selected item, there is no previous. Moving it down here made the math easier. It removed "margin" here so that when it becomes the top item, it retains the 4px margin the first element should have.